MANIFEST.in

```diff
@@ -0,0 +1,9 @@
+include LICENSE.txt
+include CONTRIBUTING.rst
+include README.rst
+
+recursive-include docs *
```

Comment:
Why not graft if you literally includes all? Also, to not match every time by *.py below, you can do same graft with global-exclude *.pyc. Which you do need to add in anyway or you'll distribute .pyc files from source tree.